templates/components/hero.html

```diff
@@ -5,5 +5,10 @@
         </figure>
 
         <h1 class="hero-title">March 6th-7th, 2025 @ Pasadena, CA</h1>
+        <h2 class="hero-subtitle">
```

Comment:
I think it should have dates for SCALE too to make it clear it's 4 days in total. A lot of people were confused last time and left without going to SCALE. At NixCon I was talking to a guy who thought that PlanetNix and SCALE were happening _at the same time_, which is wrong but not entirely obvious from the website.

Reply:
Done!